docs/query-params.md

```diff
@@ -83,14 +83,27 @@ Specify an alternate style for Firefox Accounts.
 #### When to specify
 When authenticating a user with OAuth and an alternate style is needed.
 
-## Firefox/Sync parameters
-
 #### When to specify
```

Comment:
I know this is tangential, but this now reads with consecutive `When to specify` headings. Is that intentional?